financial/bill_splitter.py

```diff
@@ -0,0 +1,88 @@
+"""
+Program to split bills among a group
+"""
+
+from typing import Any
+from typing import List as list
```

Comment:
This is a very bad idea because it overrides the builtin `list` function

Reply:
its not allowing to do List. When i commit "List" changing into 'list' :(